### ISSUE TITLE
Support Smart Invert Colors feature

### ISSUE THREAD

**Issue by warpling** [themonetizr/The-Monetizr-SDK#5]
Smart Invert Colors is a new accessibility feature in iOS 11 that helps people who are sensitive to bright light.
If a dark theme is active `accessibilityIgnoresInvertColors` should be set to yes. This needs to respond appropriately to theme changes and the toggling of this setting which can happen after app launch. Adding something like this to an overarching ViewController is a good start.

```
        #pragma clang diagnostic push
        #pragma clang diagnostic ignored "-Wpartial-availability"
        if (iOSisAtLeastVersion(11, 0, 0) && (self.style == ThemeStyleDark)) {
            [self.view setAccessibilityIgnoresInvertColors:YES];
        }
        #pragma clang diagnostic pop
```

**Comment by warpling** [comment]
This appears to be supported now, thanks!

**Comment by warpling** [comment]
For the record, smart invert is largely used so those sensitive to light can darken the screen when that's not supported. Since my app supports dark mode this isn't critical but in other apps that don't you might want to apply `accessibilityIgnoresInvertColors` to the entire view if it's already in a dark theme, or set the theme to be dark so that a dark theme doesn't become a light looking theme when smart invert is applied :)
